rotary_utils/repair.py

```diff
@@ -399,7 +423,7 @@ def iterate_linking_contig_ends(contig_record: SeqIO.SeqRecord, bam_filepath: st
             # add 1 to the total number of assembly attempts.
 
         logger.info(f'Attempting reassembly with a length threshold of {length_threshold} bp')
-        stitch_dirs.set_length_threshold(length_threshold)
+        stitch_dirs.length_threshold = length_threshold
```

Comment:
@jmtsuji Instead of calling a method, we assign it to the `length_threshold` property, which runs the internal setter.

Reply:
Cool -- so then, when `length_outdir = stitch_dirs.length_outdir` is run below, the code for the length_outdir property is run.